docs/examples/resume_crawl.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: resume-crawl
+title: Resume Crawl
```

Comment:
Could we come up with a better title? Maybe: "Resuming an existing crawl"?

Reply:
How does "Resuming a paused crawl" sound?

Reply:
Or "Pausing and resuming a crawl"